### DISCUSSION TITLE
Can Linux import a openzfsonwindows drive/pool

### DISCUSSION THREAD

**Discussion by Mokerak** [openzfsonwindows/openzfs#369] [Q&A]
I know this is probably a dumb question, but I'm about to try and turn my old PC with a openzfswindows pool/drive into a linux machine, so I'd want to just keep using the zfs pool on that. 

Would I just do the same stuff on Linux that I do in the windows command line to import the pool?

Feel free to roast me

**Comment by guenther-alka** [comment]
This is all Open-ZFS

Just do a zpool import and zpool import -D to detect ZFS pools,
then import via zpool import -F poolname
